okhttp/src/main/java/okhttp3/OkHttpClient.kt

```diff
@@ -1027,6 +1028,19 @@ open class OkHttpClient internal constructor(
       pingInterval(duration.toMillis(), MILLISECONDS)
     }
 
+    fun insecureTrustManager(hostName: String): Builder {
+      // TODO could build this up and defer to later, to avoid overwriting other changes
```

Comment:
TrustManagerBridge.Builder could short circuit to use the default only if one isn't provided. On the other hand, this functionality also fixes this weirdness

https://square.github.io/okhttp/4.x/okhttp/okhttp3/-ok-http-client/-builder/ssl-socket-factory/#trustmanagers-on-android-are-weird

Reply:
Going to refactor this, do this at the last minute when overrides are introduced.